utils/steward-cli/src/commands/command_args.rs

```diff
@@ -372,7 +390,7 @@ pub struct RemoveFromBlacklist {
 
     /// Validator index of validator list to remove
     #[arg(long, env)]
```

Comment:
```suggestion
    /// Validator indices of validator list to remove (comma separated)
    #[arg(long, env, value_delimiter = ',', num_args = 1.., value_parser = parse_u32)]
```

We can borrow similar definition of macro from `AddToBlacklist`